docs/framework/tools/mage-exe-manifest-generation-and-editing-tool.md

```diff
@@ -36,14 +36,15 @@ Mage [commands] [commandOptions]
   
 <a name="NewUpdate"></a>   
 ## New and Update Command Options  
- The following table shows the options supported by the `-New` and `-Update` commands.  
+
+The following table shows the options supported by the `-New` and `-Update` commands.  
   
 |Options|Default Value|Applies To|Description|  
 |-------------|-------------------|----------------|-----------------|  
 |**-a, -Algorithm**|sha1RSA|Application manifests.<br /><br /> Deployment manifests.|Specifies the algorithm to generate dependency digests with. Value must be "sha256RSA" or "sha1RSA.<br /><br /> Use with the "-Update" option. This option is ignored when using the "-Sign" option|  
 |**-appc, -AppCodeBase** `manifestReference`||Deployment manifests.|Inserts a URL or file path reference to the application manifest file. This value must be the full path to the application manifest.|  
 |**-appm, -AppManifest** `manifestPath`||Deployment manifests.|Inserts a reference to a deployment's application manifest into its deployment manifest.<br /><br /> The file indicated by `manifestPath` must exist, or Mage.exe will issue an error. If the file referenced by `manifestPath` is not an application manifest, Mage.exe will issue an error.|  
-|**-cf, -CertFile** `filePath`||All file types.|Specifies the location of an X509 digital certificate for signing a manifest. This option can be used in conjunction with the **-Password** option, if the certificate requires a password.|  
+|**-cf, -CertFile** `filePath`||All file types.|Specifies the location of an X509 digital certificate for signing a manifest. This option can be used in conjunction with the **-Password** option if the certificate requires a password.<br/><br/>Starting with the .Net Framework 4.6.2 SDK, which is distributed with Visual Studio and with Windows SDK, mage.exe signs manifests with  CNG as well as CAPI certificates.|  
```

Comment:
.Net -> .NET 
italicize mage.exe
remove extra spaces between with and CNG
Can't you download the SDK separately as well?